<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataStreamOpt</class>
 <widget class="QWidget" name="DataStreamOpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Stream Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="URI_label">
        <property name="text">
         <string>URI:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="URI_edit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Format_label">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Format_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>(select format)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Binary data file</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space-separated file (.ssv)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comma-separated file (.csv)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tab-separated file (.tsv)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP/IP stream</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP/IP stream</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Raw UDP/IP stream (connectionless)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Filename_label">
        <property name="text">
         <string>File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Filename_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="Filename_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Port_label">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Port_spin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>17017</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Freq_label">
        <property name="text">
         <string>Flush Rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Freq_spin">
        <property name="suffix">
         <string>Hz</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Buffer_label">
        <property name="text">
         <string>Buffer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Buffer_spin">
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>32</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="Columns_edit"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="TimeSync_label">
        <property name="text">
         <string>Time Sync. column-name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TimeSync_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionUpdateURI">
   <property name="text">
    <string>UpdateURI</string>
   </property>
   <property name="toolTip">
    <string>UpdateURI</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Filename_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Format_combo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Freq_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Port_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Buffer_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSync_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionUpdateURI</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
